iOS_SDK/OneSignalSDK/Source/OneSignalCommonDefines.h

```diff
@@ -199,6 +199,8 @@ typedef enum {GET, POST, HEAD, PUT, DELETE, OPTIONS, CONNECT, TRACE} HTTPMethod;
 #define ERROR_PUSH_NEVER_PROMPTED          -18
 #define ERROR_PUSH_PROMPT_NEVER_ANSWERED   -19
 
+#define AUTH_STATUS_EPHEMERAL 4
```

Comment:
nit, might be easier to search for and update this later if we add a comment that this is `UNAuthorizationStatusEphemeral`